frontend/src/components/pages/Landing.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import {
+  Box,
+  Center,
+  Text,
+  Stack,
+  VStack,
+  Image,
+  Button,
+  Link,
+} from "@chakra-ui/react";
+import * as Routes from "../../constants/Routes";
+
+import JumpMathLogo from "../../assets/jump-math-logo.png";
+
+const PageLinks = (): React.ReactElement => {
+  const history = useHistory();
+  const navigateTo = () => history.push(Routes.LOGIN_PAGE);
```

Comment:
once the admin and teacher login pages are built, would the flows be like this?
- when admin clicked => `ADMIN_LOGIN_PAGE`
- when teacher clicked => `TEACHER_LOGIN_PAGE`
- when student clicked => `STUDENT_PAGE`

can be a future refactor, just wanted to clarify

Reply:
yeah exactly! but since we don't have the login or student pages setup yet, don't have this logic setup yet.